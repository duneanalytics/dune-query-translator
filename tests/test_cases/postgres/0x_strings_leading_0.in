select '\xdeadbeef', '\x0010', '\x00deadbeef', '\x0000deadbeef', '\x0000000000000000000000000000000000000000'