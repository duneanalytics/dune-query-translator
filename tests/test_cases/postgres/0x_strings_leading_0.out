SELECT
  0xdeadbeef,
  0x0010,
  0x00deadbeef,
  0x0000deadbeef,
  0x0000000000000000000000000000000000000000,
  0x0f4Be9f208C552A6b04d9A1222F385785f95beAA